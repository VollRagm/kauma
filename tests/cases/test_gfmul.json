{
    "testcases": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "action": "gfmul",
            "arguments": {
                "semantic": "xex",
                "a": "ARIAAAAAAAAAAAAAAAAAgA==",
                "b": "AgAAAAAAAAAAAAAAAAAAAA=="
            }
        }
    }
}