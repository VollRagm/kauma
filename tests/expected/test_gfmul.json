{
    "responses": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "product": "hSQAAAAAAAAAAAAAAAAAAA=="
        }
    }
}